File: SendEmail/SendEmail.cs
using System.Runtime.Serialization;
using Microsoft.Phone.Tasks;

namespace WP7CordovaClassLib.Cordova.Commands
{
    public class sendEmailArgs
    {
        [DataMember]
        public string toRecipients;

        [DataMember]
        public string body;

        [DataMember]
        public string subject;
    }
    public class SendEmail : BaseCommand
    {
        public void send(string args)
        {
            sendEmailArgs myargs = JSON.JsonHelper.Deserialize<sendEmailArgs>(args);
            EmailComposeTask emailTask = new EmailComposeTask();
            emailTask.To = myargs.toRecipients;
            emailTask.Subject = myargs.subject;
            emailTask.Body = myargs.body;
            emailTask.Show();
            this.DispatchCommandResult();
        }
    }
}

File: SendEmail/SendEmail.js
function Emailer(){
    this.resultCallback = null;
}

Emailer.prototype.showEmailer = function (toRecipients, body, subject) {
    var args = {};
    if (toRecipients)
        args.toRecipients = toRecipients;

    if (body)
        args.body = body;

    if (subject)
        args.subject = subject;

    Cordova.exec(null, null, "SendEmail", "send", args);
}

Emailer.install = function () {
    console.log("install started");

    if (!window.plugins)
        window.plugins = {};

    if (!window.plugins.myEmailer) 
        window.plugins.myEmailer = new Emailer();    
}

Cordova.addConstructor(Emailer.install);